aspnet/tutorials/nano-server.rst

```diff
@@ -49,29 +49,41 @@ Once you have added your Nano Server instance to your ``TrustedHosts``, you can
 
 If you have successfully connected then your prompt will look like this ``[10.83.181.14]: PS C:\Users\Administrator\Documents>``
 
-Installing the HttpPlatformHandler Module
------------------------------------------
+Installing IIS
+--------------
 
-The :ref:`HttpPlatformHandler <http-platformhandler>` is an IIS 7.5+ module which is responsible for process management of HTTP listeners and to proxy requests to processes that it manages. At the moment, the process to install the HttpPlatformHandler Module for IIS is manual. You will need to install the latest 64-bit version of the `HttpPlatformHandler <http://www.iis.net/downloads/microsoft/HttpPlatformHandler>`_ on a regular (not Nano) machine. After installing you will need to copy the following files:
+Add the ``NanoServerPackage`` provider from the PowerShell gallery. Once the provider is installed and imported, you can install Windows packages.
 
-* *%windir%\\System32\\inetsrv\\HttpPlatformHandler.dll*
-* *%windir%\\System32\\inetsrv\\config\\schema\\httpplatform_schema.xml*
+.. code:: ps1
+
+  Install-PackageProvider NanoServerPackage
+  Import-PackageProvider NanoServerPackage
+  Install-NanoServerPackage -Name Microsoft-NanoServer-IIS-Package
+
+
+Installing the ASP.NET Core Module
+----------------------------------
+
+The :ref:`ASP.NET Core Module <http-platformhandler>` is an IIS 7.5+ module which is responsible for process management of ASP.NET Core HTTP listeners and to proxy requests to processes that it manages. At the moment, the process to install the ASP.NET Core Module for IIS is manual. You will need to install the version of the `.NET Core Windows Server Hosting bundle <https://dot.net/>`__ on a regular (not Nano) machine. After installing you will need to copy the following files:
+
+* *%windir%\\System32\\inetsrv\\aspnetcore.dll*
+* *%windir%\\System32\\inetsrv\\config\\schema\\aspnetcore_schema.xml*
 
 On the Nano machine you’ll need to copy those two files to their respective locations.
 
 .. code:: ps1
 
-  Copy-Item .\HttpPlatformHandler.dll c:\Windows\System32\inetsrv
-  Copy-Item .\httpplatform_schema.xml c:\Windows\System32\inetsrv\config\schema
+  Copy-Item .\aspnetcore.dll.dll c:\Windows\System32\inetsrv
+  Copy-Item .\aspnetcore_schema.xml c:\Windows\System32\inetsrv\config\schema
 
-Enabling the HttpPlatformHandler
+Enabling the ASP.NET Core Module
 --------------------------------
 
 You can execute the following PowerShell script in a remote PowerShell session to enable the HttpPlatformHandler module on the Nano server.
 
 .. note:: This script runs on a clean system, but is not meant to be idempotent. If you run this multiple times it will add multiple entries. If you end up in a bad state, you can find backups of the *applicationHost.config* file at *%systemdrive%\inetpub\history*.
```

Comment:
remove not meant to be
This script works on a clean machine but is not idempotent. Entries are added each time it is run. {something like} You can restore the previous state with the backups of ...